src/main/resources/meta/screen/meeting/view/meetinglist.view.json

```diff
@@ -11,8 +11,8 @@
     },
     {
       "widgetName": "meetingList",
-      "position": 2,
-      "gridWidth": 4
+      "position": 1,
```

Comment:
@achabalov there is no for changing current layout of desler-demo. It should be `2` as earlier.

Reply:
@DRITE Of course I corrected